src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,16 +2,25 @@
 
 public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-    int totalRejected = 0;
+    private int totalRejected = 0;
 
     public int reportRejected() {
-        return totalRejected;
+        return getTotalRejected();
     }
 
+    @Override
     public void countIn(int in) {
         if (!callCheck() && in > peekaboo())
-            totalRejected++;
+            setTotalRejected(getTotalRejected() + 1);
         else
             super.countIn(in);
     }
+
+    public int getTotalRejected() {
```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.